<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CXChannelEdit</class>
 <widget class="QDialog" name="CXChannelEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор канала</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Секции</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>7</number>
      </property>
      <item>
       <widget class="QListWidget" name="mSectionsList">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Имена</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>7</number>
      </property>
      <item>
       <widget class="QListWidget" name="mNamesList">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>7</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Модификатор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mModifyEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Описание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mDescriptionEdit"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="CXTouchButton" name="mAcceptButton">
     <property name="text">
      <string>Применить</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CXTouchButton</class>
   <extends>QPushButton</extends>
   <header>CXTouchButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mAcceptButton</sender>
   <signal>clicked()</signal>
   <receiver>CXChannelEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
